guides/source/asset_pipeline.md

```diff
@@ -3,561 +3,545 @@
+preserve their stable file names.
 
-Sprockets uses files named `index` (with the relevant extensions) for a special
-purpose.
+#### Excluding Directories from Digestion
 
-For example, if you have a CSS library with many modules, which is stored in
-`lib/assets/stylesheets/library_name`, the file `lib/assets/stylesheets/library_name/index.css` serves as
-the manifest for all files in this library. This file could include a list of
-all the required files in order, or a simple `require_tree` directive.
+You can exclude certain directories from the precompilation and digestion
+process by adding them to `config.assets.excluded_paths`. This is useful if, for
+example, you’re using `app/assets/stylesheets` as input to a compiler like [Dart
+Sass](https://sass-lang.com/), and you don’t want these files to be part of the
+asset load path.
 
-It is also somewhat similar to the way that a file in `public/library_name/index.html` can be reached by a request to `/library_name`. This means that you cannot directly use an index file.
-
-The library as a whole can be accessed in the `.css` files like so:
-
-```css
-/* ...
-*= require library_name
-*/
-```
-
-This simplifies maintenance and keeps things clean by allowing related code to
-be grouped before inclusion elsewhere.
-
-### Coding Links to Assets
-
-Sprockets does not add any new methods to access your assets - you still use the
-familiar `stylesheet_link_tag`:
-
-```erb
-<%= stylesheet_link_tag "application", media: "all" %>
-```
-
-If using the [`turbo-rails`](https://github.com/hotwired/turbo-rails) gem, which is included by default in Rails, then
-include the `data-turbo-track` option which causes Turbo to check if
-an asset has been updated and if so loads it into the page:
-
-```erb
-<%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+```ruby
+config.assets.excluded_paths = [Rails.root.join("app/assets/stylesheets")]
 ```
 
-In regular views you can access images in the `app/assets/images` directory
-like this:
-
-```erb
-<%= image_tag "rails.png" %>
-```
+This will prevent the specified directories from being processed by Propshaft
+while still allowing them to be part of the precompilation process.
 
-Provided that the pipeline is enabled within your application (and not disabled
-in the current environment context), this file is served by Sprockets. If a file
-exists at `public/assets/rails.png` it is served by the web server.
+Working with Propshaft
+----------------------
 
-Alternatively, a request for a file with an SHA256 hash such as
-`public/assets/rails-f90d8a84c707a8dc923fca1ca1895ae8ed0a09237f6992015fef1e11be77c023.png`
-is treated the same way. How these hashes are generated is covered in the [In
-Production](#in-production) section later on in this guide.
+From Rails 8.0, Propshaft is the default asset pipeline, so you don’t need to
+install it separately. When you create a new Rails 8 application, Propshaft is
+included by default.
```

Comment:
```suggestion
Propshaft is included by default in new Rails applications.
```

The guide is versioned so specifying that it is included from Rails 8 looks redundant.